utils/sub8_msgs/msg/PoseTwist.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
```

Comment:
This is the same as the Waypoint message I've already added. We only need to keep one or the other.